security/cmd/multicluster_ca/main.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO(myidpt): Deprecate this.
```

Comment:
Do you mean to merge this the existing istio_ca? If so, that's great. We should have only one CA unless there is strong argument.

Reply:
Yes, that's correct.